script/test-robots

```diff
@@ -3,13 +3,47 @@
 A script to run a test competition match on every robot in an submissions directory.
 """
 import sys
+import time
 import shutil
 import argparse
 import tempfile
 import subprocess
 from typing import List
 from pathlib import Path
 
+EXCESS_SUBPROCESS_TIMEOUT = 10
```

Comment:
A comment here about the unit (or baking that into the constant name) might be useful.

Reply:
e59a4d998a62a0dea0e8615e63ccbe3f33f4ba9a